app/angular/package.json

```diff
@@ -41,19 +41,18 @@
     "dotenv-webpack": "^1.5.5",
     "global": "^4.3.2",
     "html-loader": "^0.5.5",
-    "html-webpack-plugin": "^2.30.1",
+    "html-webpack-plugin": "^3.0.7",
     "json5": "^0.5.1",
     "markdown-loader": "^2.0.2",
     "node-sass": "^4.8.3",
     "raw-loader": "^0.5.1",
     "react": "^16.0.0",
-    "react-dev-utils": "^5.0.0",
+    "react-dev-utils": "zanettin/react-dev-utils",
```

Comment:
Is this something we could revert back now?

Reply:
https://github.com/facebook/create-react-app/pull/4077 is still open